### PR TITLE
Fix mvn clean package build error, #24.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: initializr-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -1371,6 +1371,9 @@ initializr:
     - name: Jar
       id: jar
       default: true
+    - name: Pom
```
Comment (yaweiw) [inline initializr-service/src/main/resources/application.yml:1374]: why is this needed? Jar or War is UI elements?

**Comment by Incarnation-p-lee** [comment]
close this PR for just modify the template. The `		BeanWrapperImpl bean = new BeanWrapperImpl(request);` from code seems not works well.
